### PR TITLE
CIP 46 - Processes Around Granda Mento Exchanges

### PR DESCRIPTION
**Pull request by tkporter**
A CIP defining processes around Granda Mento exchanges.

### CODE REVIEW

**Comment by tobikuhlmann** [comment]
Looks very clear to me! 

Only thing I've been thinking about while reading were the governance approval issues. I know that MultiSig members are active community members, should they track the forum / discord for new proposals or should the exchange proposer actively reach out to make sure the approvers are aware of an active exchange proposal?

Could imagine a @granda_mento_approvers discord handle or e-mail would be helpful

**Comment by tkporter** [comment]
@tobikuhlmann good idea, I like the discord handle idea, that way it's a little more in the public eye as well.

File: CIPs/cip-xxxx.md
Comment (Pinotio) [inline CIPs/cip-xxxx.md:38]: I think there would be no <buy amount> for the <buy token> as the rate is not known yet?

instead of <buy token>, you could write <buy stabletoken> or <buy cStable>

File: CIPs/cip-xxxx.md
Comment (Pinotio) [inline CIPs/cip-xxxx.md:72]: So the proposer has to hold CELO for one week? Wouldn't the fluctuations in price during that period potentially outweigh the benefit of lower slippage with Granda Mento? I guess we'll see if anyone uses Granda Mento.

File: CIPs/cip-xxxx.md
Comment (Pinotio) [inline CIPs/cip-xxxx.md:125]: typo "whether an approver multisig signer..."

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:5]: Needs a discussion link. Please create a github issue ticket

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:7]: It's not a Meta CIP, a Meta CIP contains multiple CIPs in it. This is more of an Informational CIP given its around processes.

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:9]: delete

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:10]: delete

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [reply CIPs/cip-xxxx.md:72]: @Pinotio let's reserve content discussions for the github discussions link as soon as @tkporter creates one

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:42]: Would help to a have a link to a template forum post as well.

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [inline CIPs/cip-xxxx.md:76]: Who are the signers of the approver multisig?

Comment (YazzyYaz) [reply CIPs/cip-xxxx.md:76]: It should be specified.

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

**Comment by YazzyYaz** [comment]
Needs some changes. Am I correct that it's now ready for review and being merged once feedback is implemented? If so, maybe update the [DRAFT WIP DO NOT MERGE] so that it's not confusing.

File: CIPs/cip-0046.md
Comment (martinvol) [inline CIPs/cip-0046.md:48]: Does this accept anonymous  or pseudoanonymous?

File: CIPs/cip-0046.md
Comment (martinvol) [inline CIPs/cip-0046.md:63]: I guess this question would make many players uncomfortable or goes against their internal privacy rules, can they alternatively disclosure the date of the purchase?

File: CIPs/cip-0046.md
Comment (martinvol) [inline CIPs/cip-0046.md:93]: We can make it explicit that this proposal must only have a `cancelExchangeProposal` cal for the Celo Governance Approver multisig to approve it.

File: CIPs/cip-0046.md
Comment (tkporter) [reply CIPs/cip-0046.md:48]: Up to the community! I think it should be left open ended

File: CIPs/cip-xxxx.md
Comment (tkporter) [reply CIPs/cip-xxxx.md]: This feels like something that should be in the CGP to set the approver multisig, not in a CIP that sets out general processes. The multsig may change from a later governance proposal and it doesn't feel appropriate to have to update a CIP to reflect on-chain state.

File: CIPs/cip-xxxx.md
Comment (tkporter) [reply CIPs/cip-xxxx.md]: From https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0000.md:
```
* A Meta CIP describes a process surrounding Celo or proposes a change to (or an event in) a process. Meta CIPs are like Standards Track CIPs but apply to areas other than the Celo protocol itself. They may propose an implementation, but not to Celo’s codebase; they often require community consensus; unlike Informational CIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include code of conduct updates, procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Celo development. Any Meta CIP is also considered a Process CIP.
* An Informational CIP describes a Celo design issue, or provides general guidelines or information to the Celo community, but does not propose a new feature. Informational CIPs do not necessarily represent Celo community consensus or a recommendation, so users and implementers are free to ignore Informational CIPs or follow their advice.
```

I don't see anything about a Meta CIP containing multiple CIPs?
This CIP feels to me like it falls somewhere in the category of `procedures, guidelines, changes to the decision-making process`. Users are not really free to ignore these processes and I believe this CIP requires Celo community consensus.

File: CIPs/cip-0046.md
Comment (tkporter) [reply CIPs/cip-0046.md:63]: Just added a note about exchangers being encouraged to answer all questions for transparency, and if they wish not to answer some they should explain why

File: CIPs/cip-xxxx.md
Comment (YazzyYaz) [reply CIPs/cip-xxxx.md]: Good point, sounds good

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**
